stock_location_address_purchase/tests/test_purchase_buy_rule.py

```diff
@@ -10,9 +10,8 @@ def test_no_destination(self):
         company = self.env.ref("base.main_company")
         # Update company with Purchase Lead Time
         company.write({"po_lead": 3.00})
-        date_planned = fields.Datetime.to_string(
-            fields.datetime.now() + timedelta(days=10)
-        )
+        date_planned = fields.datetime.now() + timedelta(days=10)
```

Comment:
@nitohu Could you add a test case to highlight this change ?

IMHO, as Datetime field is now a datetime, this is more correct:

```suggestion
        date_planned = fields.Datetime.now() + timedelta(days=10)
```

Reply:
Hello, I have added a test for a thrown error when calling this function with a string as it's argument on line 25-31 & 50-55 and changed datetime to Datetime.